servicecontrol/troubleshooting.md

```diff
@@ -105,3 +105,11 @@ The internal *FailedMessageRetries* collection must be purged in order to restor
 1. Select the "FailedMessageRetries" collection in the left tree
 1. Delete all documents in the collection
 1. Stop maintenance mode
+
+## Microsoft Defender SmartScreen blocks installer 
```

Comment:
```suggestion
## SmartScreen blocks the installer
```

Let's stay away from trying to qualify the name, as per [Wikipedia](https://en.wikipedia.org/wiki/Microsoft_SmartScreen) —

> SmartScreen (officially called Windows SmartScreen, Windows Defender SmartScreen and SmartScreen Filter in different places) is a... (fill in appropriate term here)

![this-is-fine](https://user-images.githubusercontent.com/677704/102587031-7b0fa280-410b-11eb-97a1-c7accebe859b.gif)